### ISSUE TITLE
[bug/needsfix] Write new instruction set prompts w/ Mistral/Mixtral support

### ISSUE THREAD

**Issue by hyd3rs** [spongedsc/pathways#105]
# Description
At the moment, all current prompts for personas are written/tuned for Llama 3. However, with the introduction of Integrations and the subsequent move of various parts of the ML stack to models based on Mistral/Mixtral, prompts for "personality-heavy" personas are now broken or severely altered. All affected personas will need to be rewritten to support Mistral/Mixtral models.

Affected personas are as follows:
- `spongeass`

**Comment by artifishvr** [comment]
@daniwasonline #95 exists, would that prompt perform better on mistral ai models?

**Comment by daniwasonline** [comment]
yes it probably would be, i'm testing out mistral now so we'll have to see
